### PR TITLE
Create a custom Artemis Operator image to be used temporarily

### PR DESCRIPTION
**Pull request by tommaso-borgato**
## Description
See https://github.com/Intersmash/intersmash/issues/32
Basically this MR introduces a temporary ActiveMQ operator catalog to be used while we wait that the ActiveMQ operator is added to the OpenShift catalog

## Type of change
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] Pull Request contains a description of the changes
 - [x] Pull Request does not include fixes for multiple issues/topics
 - [x] Code is self-descriptive and/or documented
 - [ ] I have implemented unit tests to cover my changes
 - [x] I tested my code in OpenShift

### CODE REVIEW

File: global-test.properties [hunk 1]
```diff
@@ -18,6 +18,12 @@ intersmash.wildfly.operators.index_image=quay.io/operatorhubio/catalog:latest
 intersmash.wildfly.operators.package_manifest=wildfly
 intersmash.wildfly.operators.channel=alpha
 
+# ActiveMQ community operator settings
+intersmash.activemq.operators.catalog_source=intersmash-activemq-operator-index
+intersmash.activemq.operators.index_image=quay.io/jbossqe-eap/intersmash-activemq-operator-catalog:v1.0.11
```
Comment (fabiobrz) [inline global-test.properties:23]: Just a question here, is the version you used (v1.0.11) derived from some Artemis operator tag or did you reate it randomly (I assume not)?

**Review by fabiobrz — COMMENTED** [review]
HI @tommaso-borgato and thanks for this PR, great job.
I've dropped one question regarding the version you used for the custom bundle/index.

File: global-test.properties (hunk 1 above)
Comment (tommaso-borgato) [reply global-test.properties:23]: it was the default version used when building the upstream repo

Comment (fabiobrz) [reply global-test.properties:29]: I see, thanks. Which branch have we built?

Comment (tommaso-borgato) [reply global-test.properties:29]: main, now I've rebuilt everything using the latest tag

**Pull request merged**